packages/conversation/tests/rest/v1/webhooks-events/webhooks-events.steps.ts

```diff
@@ -0,0 +1,269 @@
+import { Given, Then, When } from '@cucumber/cucumber';
+import { ConversationCallbackWebhooks, Conversation } from '../../../../src';
+import assert from 'assert';
+import { IncomingHttpHeaders } from 'http';
+
+const APP_SECRET = 'CactusKnight_SurfsWaves';
+let conversationCallbackWebhook: ConversationCallbackWebhooks;
+let rawEvent: any;
+let event: Conversation.ConversationWebhookEvent;
+let formattedHeaders: IncomingHttpHeaders;
+
+const processEvent = async (response: Response) => {
+  formattedHeaders = {};
+  response.headers.forEach((value, name) => {
+    formattedHeaders[name.toLowerCase()] = value;
+  });
+  rawEvent = await response.text();
+  rawEvent = rawEvent.replace(/\s+/g, '');
+  event = conversationCallbackWebhook.parseEvent(JSON.parse(rawEvent));
+};
+
+Given('the Conversation Webhooks handler is available', () => {
+  conversationCallbackWebhook = new ConversationCallbackWebhooks(APP_SECRET);
+});
+
+Then('the Conversation event header contains a valid signature', () => {
+  assert.ok(conversationCallbackWebhook.validateAuthenticationHeader(formattedHeaders, rawEvent));
+});
+
+When('I send a request to trigger a "CAPABILITY" event', async () => {
+  const response = await fetch('http://localhost:3014/webhooks/conversation/capability-lookup');
+  await processEvent(response);
+});
+
+Then('the Conversation event describes a "CAPABILITY" event type', () => {
+  const capabilityEvent = event as Conversation.CapabilityEvent;
+  assert.ok(capabilityEvent.capability_notification);
+  const expectedTrigger: Conversation.WebhookTrigger = 'CAPABILITY';
```

Comment:
The trigger value is a value created by SDK and not present from payload.
So validation against `capabilityEvent.capability_notification` field seems OK to me
But validating again the trigger value seems more related to a unit tests not a e2e test
What do you think about N?